ui/component/or-rules/src/or-rule-list.ts

```diff
@@ -615,6 +616,21 @@ export class OrRuleList extends translate(i18next)(LitElement) {
 
     }
 
+    protected _onSettingsClicked() {
+        if (this._globalRulesets) {
+            showOkCancelDialog("Global rules settings", html`
```

Comment:
Can you make the string translatable using `i18next.t(...)` which is used elsewhere?